supervisord/datadog_checks/supervisord/supervisord.py

```diff
@@ -122,32 +122,63 @@ def check(self, instance):
         if not isinstance(proc_regex, list):
             raise Exception("'proc_regex' should be a list of strings. e.g. %s" % [proc_regex])
 
+        proc_regex_exclude = instance.get('proc_regex_exclude', [])
+        if not isinstance(proc_regex_exclude, list):
+            raise Exception("'proc_regex_exclude' should be a list of strings. e.g. %s" % [proc_regex_exclude])
+
         proc_names = instance.get('proc_names', [])
         if not isinstance(proc_names, list):
             raise Exception("'proc_names' should be a list of strings. e.g. %s" % [proc_names])
 
+        proc_names_exclude = instance.get('proc_names_exclude', [])
+        if not isinstance(proc_names_exclude, list):
+            raise Exception("'proc_names_exclude' should be a list of strings. e.g. %s" % [proc_names_exclude])
+
+        # Compile status mapping
+        status_mapping = DD_STATUS
```

Comment:
Could you copy `DD_STATUS` here (deepcopy if necessary)? This is because it's a constant, but we're modifying it. later on.

Reply:
Fixed: https://github.com/jcolfej/datadog-integrations-core/commit/e5d1002ff6c53e07c4c68995c32faa3a0af4d0aa